src/main/java/frc/robot/subsystems/BuddyBalanceSubsystem.java

```diff
@@ -177,5 +178,13 @@ public void periodic() {
     positionEntry.setDouble(AbsoluteEncoder.getPositionRadians(encoder));
     encoderCalculatedAngle = unifiedPIDController.calculate(getDesiredAngle());
     SmartDashboard.putNumber("Buddy Balance Angle", Units.radiansToDegrees(AbsoluteEncoder.getPositionRadians(encoder)));
+    // to prevent the world from ending (cam breaking swerve modules)
+    if ((AbsoluteEncoder.getPositionRadians(encoder) > Math.PI) || (AbsoluteEncoder.getPositionRadians(encoder) < 0)) {
+      rightMotor1.set(0);
+      rightMotor2.set(0);
+      leftMotor1.set(0);
+      leftMotor2.set(0);
+      unifiedPIDController.setSetpoint(Units.degreesToRadians(90));
+    }
```

Comment:
Like we discussed, I would prefer that the check is done right before calculate, so that we don't have a chance to move out of our safe zone at all.  
Our safe zone should be defined with constants, so that it is more apparent what is happening.  RIght now you are matching to the same value as the raised and deployed angle, yes?  

Why are we setting the setpoint at all at this point?  To move it back inside of the safe zone?
We should think about this more.  Since, this is a one time in a match action, it might be acceptable to just stay at wherever we landed near 180degrees.  

If not, then we shouldn't change the setpoint, but figure out how to move the motors back the way they came.